extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-preview.js

```diff
@@ -782,6 +789,9 @@ export class AmpStoryDevToolsTabPreview extends AMP.BaseElement {
     dialog.querySelector(
       '.i-amphtml-story-dev-tools-help-search-preview-link'
     ).href += this.storyUrl_;
+    dialog.querySelector(
+      '.i-amphtml-story-dev-tools-help-page-experience-link'
+    ).href += encodeURIComponent(this.storyUrl_);
```

Comment:
Nit: maybe build the entire URL from here to reduce the risk of a malformed URL?